config/environment.js

```diff
@@ -82,5 +82,21 @@ module.exports = function(environment) {
 
   }
 
+  /* global process*/
+  try {
+    let additionalReportExports = JSON.parse(process.env.timed_report_export)
+    if (additionalReportExports) {
+      ENV.APP.REPORTEXPORTS = ENV.APP.REPORTEXPORTS.concat(
+        additionalReportExports
+      )
+    }
+  } catch (e) {
+    /* eslint no-console: ["error", { allow: ["error"] }] */
```

Comment:
There are easier ways to write this, for example:
`// eslint-disable-next-line no-console`

See http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments